src/components/PerPageSelect/PerPageSelect.tsx

```diff
@@ -0,0 +1,33 @@
+import { FC } from 'react';
+
+interface Props {
+  perPage: number;
+  handlePerPageSelect: (event: React.ChangeEvent<HTMLSelectElement>) => void;
+}
+
+export const PerPageSelect: FC<Props> = ({ perPage, handlePerPageSelect }) => {
+  return (
+    <div className="form-group row">
+      <div className="col-3 col-sm-2 col-xl-1">
+        <select
+          data-cy="perPageSelector"
+          id="perPageSelector"
+          className="form-control"
+          value={perPage}
+          onChange={handlePerPageSelect}
+        >
+          <option value="3">3</option>
+          <option value="5">5</option>
+          <option value="10">10</option>
+          <option value="20">20</option>
```

Comment:
I created an array with options. Find that it better, because we can add an option like one number to array, and not modifie options by adding element. 
and here we just map this array of potential options